alpha/lib/model/DeliveryProfilePeer.php

```diff
@@ -588,6 +589,53 @@ public static function retrieveByTypeAndPks($pks, $type)
 		
 		return DeliveryProfilePeer::doSelect($criteria);
 	}
+
+	public static function getDeliveriesByIds($entry, $deliveryIds, Partner $partner, DeliveryProfileDynamicAttributes $deliveryAttributes, $checkSecured = true)
+	{
+		if (count($deliveryIds))
+		{
+			$isSecured = $checkSecured ? self::isSecured($partner, $entry) : false;
+			$isLive = $entry->getType() == entryType::LIVE_STREAM;
+			return self::getDeliveryByIds($deliveryIds, $partner, null, $deliveryAttributes, null, $isSecured, $isLive);
+		}
+
+		return array();
+	}
+
+	public static function getDefaultDeliveriesFilteredByStreamerTypes($entry, Partner $partner, $excludedStreamerTypes, $checkSecured = true)
```

Comment:
Same comment as above ... I don't se this function used.
Also we already have methods in this class that implement this logic try repurposing them instead of adding new once.